develop/TOOLS/TOONuGet/NuGet_Naming_Conventions.md

```diff
@@ -0,0 +1,96 @@
+---
+uid: NuGet_Naming_Conventions
+---
+
+# NuGet naming conventions
+
+> [!IMPORTANT]
+> This section might include some information that is only applicable to Skyline employees and/or links that are only accessible to Skyline employees.
+
+Below rules are mainly focused for NuGets that are published to nuget.org. It is adviced to use the same rules for internal NuGets as well as that would make it easy transferable to nuget.org when necessary.
+
+## Mandatory rules
+
+- NuGet must have a name that starts with 'Skyline.DataMiner'.
+- NuGet must include a license file when published to nuget.org
+- NuGet must be signed otherwise this will be blocked by nuget.org
+    - When creating the NuGet via [SLC SE Repository Manager](xref:TOOSLCSERepositoryManager), the pipeline will take care of the signing and publishing.
+- [Mandatory metadata](#mandatory-metadata)
+
+<br/>
+
+- If there is a dependency on a DataMiner DLL (from the Skyline DataMiner/Files folder), use the corresponding NuGet (Skyline.DataMiner.Files.XXX).
+    - If it is not there, contact the [Data-Acquisition](mailto:support.data-acquisition@skyline.be) domain. They can then include this DLL in the automatic NuGet creation from DataMiner versions.
+- If there is a dependency on multiple DataMiner DLLs that are default included in a QAction/Exe, then make use of the DevPacks (Skyline.DataMiner.Dev.XXX)
+
+<br/>
+
+- Use the syntax: 'Skyline.DataMiner.\[Categories\].\[Name\]'. Optionally at the end you could have '.Protocol', '.Automation', '.Common', ... to specify on where it can be used (usage of SLManagedScripting, SLManagedAutomation, ...)
```

Comment:
- Use the syntax: 'Skyline.DataMiner.\[Categories\].\[Name\]'. Optionally at the end you could have '.Protocol', '.Automation', '.Common', ... to specify on where it can be used (usage of SLManagedScripting, SLManagedAutomation, ...)
 => 
- Use the syntax: 'Skyline.DataMiner.<Category>.<Name>'. Optionally at the end you could have '.Protocol', '.Automation', '.Common', ... to specify where it can be used (usage of SLManagedScripting, SLManagedAutomation, ...)